lib/display.go

```diff
@@ -207,6 +207,11 @@ func highlightAlgorithm(sigAlg simpleSigAlg) string {
 	return color.SprintFunc()(algString(sig))
 }
 
+// timeString formats a time in UTC with minute precision, in the given color.
+func timeString(t time.Time, c *color.Color) string {
+	return c.SprintfFunc()(t.Format("2006-01-02 15:04Z"))
```

Comment:
This will always print Z, even if the date is not in UTC: https://play.golang.org/p/vkxNRfGRtI

You probably want "Z07" or "Z0700" instead.